### PR TITLE
chore: add volunteer tweetandcode

### PR DESCRIPTION
**Pull request by tweetandcode**

### CODE REVIEW

File: data/users/tweetandcode.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Vivek S",
+    "bio": "Fullstack | Web Dev | Javascript",
```
Comment (codewithvoid) [inline data/users/tweetandcode.json:3]: pick one title. recommendation: `Full Stack Developer`

File: data/users/tweetandcode.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Vivek S",
+    "bio": "Fullstack | Web Dev | Javascript",
+    "image": "https://avatars.githubusercontent.com/u/119622641?s=400&u=f3a76c030da69ff20d4cdb8986af793f16ce6e40&v=4",
```
Comment (codewithvoid) [inline data/users/tweetandcode.json:4]: use simpler url `https://avatars.githubusercontent.com/u/119622641?v=4`

File: data/users/tweetandcode.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Vivek S",
+    "bio": "Fullstack | Web Dev | Javascript",
+    "image": "https://avatars.githubusercontent.com/u/119622641?s=400&u=f3a76c030da69ff20d4cdb8986af793f16ce6e40&v=4",
+    "categories": [
```
Comment (codewithvoid) [inline data/users/tweetandcode.json:5]: categories need to be splitted in multiple entries. E.g

```json
  "categories": [
    {
      "slug": "programming | software-development | web-development",
      "topics": [
        "JavaScript"
      ]
    },
    {
      "slug": "web-development",
      "topics": [
        "CSS",
        "HTML"
      ]
    }
  ],
```

**Comment by codewithvoid** [comment]
Thanks for volunteering @tweetandcode . Added some comments. PTAL.

**Comment by tweetandcode** [comment]
@codewithvoid fixed those.

**Comment by codewithvoid** [comment]
Perfect. Thanks and Welcome @tweetandcode

**Pull request merged**